tests/framework/installer/ceph_installer.go

```diff
@@ -96,47 +89,46 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 		assert.NoError(h.T(), err)
 	}
 
-	err = h.k8shelper.CreateNamespace(namespace)
-	if err != nil {
-		if kerrors.IsAlreadyExists(err) {
-			logger.Warningf("Namespace %q already exists!!!", namespace)
-		} else {
-			return fmt.Errorf("failed to create namespace %q. %v", namespace, err)
-		}
+	// The operator namespace needs to be created explicitly, while the cluster namespace is created with the common.yaml
+	if err := h.k8shelper.CreateNamespace(h.settings.OperatorNamespace); err != nil {
+		return err
 	}
 
-	// disable admission controller for Kubernetes version older than v1.16.0 and for upgrade test as api version v1 require minimum v0.7 controller runtime and upgrade test still using
-	// older version of controller runtime.
-	if !utils.IsPlatformOpenShift() && namespace != "upgrade-ns-system" {
-		err = h.startAdmissionController(namespace)
-		if err != nil {
-			return fmt.Errorf("Failed to start admission controllers: %v", err)
-		}
+	// Create the namespace and RBAC before starting the operator
+	_, err = h.k8shelper.KubectlWithStdin(h.Manifests.GetCommon(), createFromStdinArgs...)
+	if err != nil {
+		return fmt.Errorf("Failed to create rook-operator pod: %v ", err)
```

Comment:
Since `"github.com/pkg/errors"` is now imported and used in this file I'd use it everywhere.